recipes/inforna/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+make clean
+make all VIENNA=$PREFIX
+
+mv INFO-RNA-2.1.2 $PREFIX/bin
```

Comment:
Including the dash depends on how the authors named it. Having both in and using a symlink is fine to me too. This is also in concordance with linux conventions AFAIK